docs/source/monitoring.md

```diff
@@ -1,76 +1,130 @@
+```
+
+For details, refer to the below command:
+```console
+$ kubectl explain scylladbmonitorings.scylla.scylladb.com/v1alpha1
+```
+
+## Deploy managed monitoring
+
+**Note**: as of v1.8, ScyllaDBMonitoring is in alpha state and is susceptible to backwards incompatible changes.
+
+### Requirements
+
+Before you can set up your ScyllaDB Monitoring, you must install the Scylla Operator in your Kubernetes cluster 
+and have an instance of ScyllaCluster running. For details see:
+* [Deploying Scylla on a Kubernetes Cluster](generic.md)
+* [Deploying Scylla stack using Helm Charts](helm.md)
+
+The managed monitoring setup also requires HAProxy Ingress and Prometheus Operator deployed in your Kubernetes cluster.
+You can deploy them using the provided third party examples.
+
+#### Deploy Prometheus Operator
+Deploy Prometheus Operator using kubectl:
+```console
+$ kubectl -n prometheus-operator apply --server-side -f examples/third-party/prometheus-operator
+```
+
+##### Verify Prometheus Operator status
+```console
+$ kubectl -n prometheus-operator get deployments.apps/prometheus-operator
+NAME                  READY   UP-TO-DATE   AVAILABLE   AGE
+prometheus-operator   1/1     1            1           5s
+```
+
+#### Deploy HAProxy Ingress
+Deploy HAProxy Ingress using kubectl:
+```console
+$ kubectl -n haproxy-ingress apply --server-side -f examples/third-party/haproxy-ingress
+```
+
+##### Verify HAProxy Ingress status
+```console
+$ kubectl -n haproxy-ingress get deployments.apps/haproxy-ingress
+NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+haproxy-ingress   1/1     1            1           5s
+```
+
+Before you begin, update the `endpointSelector` in `examples/monitoring/v1alpha1/scylladbmonitoring.yaml` with a label 
+matching your ScyllaCluster instance.
+
+### Deploy ScyllaDBMonitoring
+
+Deploy the monitoring setup using kubectl:
+```console
+$ kubectl apply --server-side -f examples/monitoring/v1alpha1/scylladbmonitoring.yaml
+```
+
+The Operator should pick up the object's creation and create all the necessary resources for you.
+
+#### Verify ScyllaDBMonitoring status
+```console
+$ kubectl get ScyllaDBMonitoring example
+NAME      AGE
+example   60s
+```
+
+#### Verify Prometheus status
+```console
+$ kubectl get statefulset.apps/prometheus-example
+NAME                 READY   AGE
+prometheus-example   1/1     68s
+```
+
+#### Verify Grafana status
+```console
+$ kubectl get deployments.apps/example-grafana
+NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+example-grafana   1/1     1            1           68s
+```
+
+#### Verify Grafana admin credentials secret
+Among other resources, the Operator creates a secret containing the admin credentials for you Grafana deployment.
+```console
+$ kubectl get secrets/example-grafana-admin-credentials
+NAME                                TYPE     DATA   AGE
+example-grafana-admin-credentials   Opaque   2      68s
+```
+
+### Access Grafana locally
```

Comment:
I think this should be a variant when you don't want to use/install haproxy, or expose it to the world

Reply:
Do you suggest removing it or adding it as an alternative?

Reply:
I think we'd primarily describe a way to use the ingress, this would be an alternative, if needed